src/Test/TestCombatMods.java

```diff
@@ -128,16 +127,14 @@ private boolean testRetribution()
     BattleSummary normalBA = CombatEngine.simulateBattleResults(infB, infA, testMap, 7, 4);
 
     venge.modifyAbilityPower(42); // juice up
-    performGameAction(new GameAction.AbilityAction(venge, venge.getReadyAbilities().get(1)), testGame); // activate Retribution
+    venge.getReadyAbilities().get(1).activate(testMap); // activate Retribution
 
     // ...and after power
     BattleSummary retribAB = CombatEngine.simulateBattleResults(infA, infB, testMap, 7, 4);
     BattleSummary retribBA = CombatEngine.simulateBattleResults(infB, infA, testMap, 7, 4);
 
     // Check that Venge's Retribution works properly without breaking things (other than balance)
     boolean testPassed = true;
-    testPassed &= validate(infB.model.getDamageRatio() > 110, "    Retribution didn't buff offense.");
-    testPassed &= validate(infB.model.getDefenseRatio() < 100, "    Retribution didn't reduce defense.");
```

Comment:
Why delete? Aren't these effects still part of Retribution?

Reply:
Yes, they are in Retribution... but as `UnitModifier`s, so the stat change won't show up until combat calculation happens.
I could manually invoke the combat internals to inspect the stat changes there, but it's just as good to observe the damage output does what I expect (and that test's still in there).

It might be a good idea to get rid of these functions and just use raw instance variables at this point, though, for two reasons:
My original intent for them is OBE - hiding the actual value behind g/setters was to enforce relative changes only, so that ability stat changes would be more likely to be cleanly reversible... but we're not touching the stat on `UnitModel` anymore, so the base stat should represent the D2D and nothing else. (It might be a good idea to move those functions to `UnitContext` and/or `StrikeParams` to enforce addition/subtraction there?)
Having a getter kinda implies (now) that the `UnitModifier`s will be consulted to produce a reasonably-close-to-correct value, but that's not what happens here. I don't really want to change that, either, as I don't want to add more spaghetti to `CombatEngine` than I need to for the actual functionality I want to support.

Reply:
If the model value is now only ever going to be the D2D, then it makes sense to remove the associated calls and use it directly. I'd probably also rename the fields to clarify that they are the D2D value, not necessarily the current value.

Reply:
I assume this is fair to kick out to the next PR, since it directly pertains?

Reply:
Yeah, that's fair.